src/index.js

```diff
@@ -1,12 +1,29 @@
 import * as functions from 'firebase-functions'
+import _ from 'lodash'
+import request from 'superagent'
 
-// A function triggered by onWrite event to /projectQueries directory
 export const onProjectQuery = functions.database.ref('/projectQueries').onWrite(event => {
   const data = event.data.val()
-  console.log(data)
-})
+  let newestTimestamp = 0
+  _.mapKeys(data, (value, key) => {
```

Comment:
You want to get the latest key (which is a timestamp)? .keys().sort()? rather than mutating?